### PR TITLE
map feature first draft/help wanted to code review and issue with refresh

### PR DESCRIPTION
**Pull request by jnovak79**
Added the initial working implementation of the retail search feature. Currently able to search for stores based off of location (City and/or State). However, need to fix issue with the route causing an issue on refresh. 

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant screenshots of the feature if applicable.

## Type of Change

Please label your pull request as appropriate for the type of change it makes and remove labels that are not relevant below:

- bug: something isn't working
- build: affects the application build process or repository
- documentation: improvements or additions to documentation
- duplicate: this issue or pull request already exists
- enhancement: new feature or request
- good first issue: good for newcomers
- help wanted: extra attention is needed
- invalid: this doesn't seem right
- question: further information is requested
- wontfix: this will not be worked on

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.md) for this project.
- [ ] My code follows the code style of this project.
- [ ] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [ ] If applicable, I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Reviewers

Please tag one or more reviewers who you believe are best suited to review this code.

## Additional Notes

Please add any additional notes or context about this pull request here.

### CODE REVIEW

File: server/index.ts
Comment (epun21403) [inline server/index.ts:1]: On line 26 do you need err if you are not using it in the console.log on line 27?

File: client/src/components/pages/SellersPage.tsx
Comment (epun21403) [inline client/src/components/pages/SellersPage.tsx:1]: On line 70, do you need err if you're not using it in the console.log on line 71?

On line 109, do you need err if you're not using it in the console.log on line 110?

**Review by epun21403 — COMMENTED** [review]
Some change reccommendations

**Comment by epun21403** [comment]
Otherwise LGTM 👁 👄 👁

**Review by epun21403 — COMMENTED** [review]
Maybe needs typescript changes to inputs

**Review by afrodynamic — CHANGES_REQUESTED** [review]
LGTM, will add a commit to address the comment and add typing/consistency fixes

**Comment by afrodynamic** [comment]
Closing to fix merge conflict, resolving in new PR

**Pull request closed without merging**